data/mods/Generic_Guns/firearms/rifle.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "rifle_assault",
+    "copy-from": "ar10",
+    "type": "GUN",
+    "name": "service rifle",
+    "ammo": "ammo_rifle",
+    "description": "Borne from years of refinement .  The products of decades of improvement, rifle such as this are handy, reliable, and adaptable.   Short of large creatures and light vehicles, this should take care of most of your problems out to several hundred meters.",
```

Comment:
```suggestion
	    "description": "Borne from years of refinement.  The products of decades of improvement, rifle such as this are handy, reliable, and adaptable.   Short of large creatures and light vehicles, this should take care of most of your problems out to several hundred meters.",
```

Reply:
@I-am-Erk oh right, I never finished this description...

Reply:
Just went ingame to check all the itemgroups. going to note down items that need migrating

luty smg .40
luty smg .45
luty smg 9mm
winchester m37
Browning auto 5
CZ-75
Glock 31
hi-point c9
Kel tec p32
Kel-tec p3AT
kel-tec pf-9
marlin 1895
mossberg 930 spx
PT 603
remington 1100 competition
remington 700
remington 870 express
ruger LCP
Sig p320
Walther p22
Walter p38
AT 4
m74 rocket clip
m110A1
Glock 18c
M17
m1911A1
Mk 23 MOD
mcmillan tac-50
mossberg 590A
remington 870 MCS
Magnum research BFR
Saiga-410
Taurus raging judge magnum
USAS 12
Uberti cattleman
Browning hi-power .40
Walther CCP
Walther PPQ
S&W model 10
Walther ppq 9mm
m45A1
Bond arms derringer
Ar-15
Hk417
1874 sharps
1887 bootleg shotgun
Colt lighting .45 carbine
m1987 trench gun
mac-11
Tavor TS12
IWI tavor x95
ar-10
Marlin 1895 SBL
Remington 700
Remington ACR .300
Cobray street sweeper
Franchi spas-12
PTR 603
H&K mp5k-pdw
H&K mpsd
.45-70 LFN
usas-12 drum magazine
saiga-410 drum magazine
usas-12 box magazine
saiga-410 box magazine
AF2011A1 magazine
.300ac blackout conversion kit
.410 000


ammo_shotgun_obscure spawns nothing
guns_launcher_grenade spawns nothing
guns_launcher_improvised spawns nothing
guns_smg_obscure spawns nothing
mags_shotgun_common spawns nothing

That's all the ones i could find easily, there might still be more.